pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/ExtensibleLoadManagerImpl.java

```diff
@@ -106,6 +107,8 @@ public class ExtensibleLoadManagerImpl implements ExtensibleLoadManager {
 
     private static final long MONITOR_INTERVAL_IN_MILLIS = 120_000;
 
+    private static final String ELECTION_ROOT = "/loadbalance/extension/leader";
```

Comment:
If the same leader plays both old and new LM leader, it will have problems. When lookup the LM system topic, it might redirect to the old leader broker, but it doesn't start the new LM in this broker, so it will redirect to the new LM broker again...

Reply:
I see.